doc/source/management/data_at_rest_encryption.rst

```diff
@@ -220,18 +192,47 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. _ps.data-at-rest-encryption.redo-log:
+
+Redo Log Encryption
+================================================================================
+
+:Availability: This feature is **Alpha** quality
+
+InnoDB redo log encryption is enabled by setting the variable
+``innodb_redo_log_encrypt``. This variable has two values: ``MASTER_KEY`` and
+``KEYRING_KEY``.
+
+``MASTER_KEY`` uses the InnoDB master key to encrypt unique keys for each log
+file in the redo log header. ``KEYRING_KEY`` uses the ``percona_redo`` versioned
+key from the keyring. When ``innodb_redo_log_encrypt`` is set to
+``KEYRING_KEY``, each new redo log file is encrypted with the latest
+``percona_redo`` key from the keyring.
+
+System variables
+--------------------------------------------------------------------------------
+
+.. variable:: innodb_redo_log_encrypt
```

Comment:
What about the other variables? Undo log encryption, scrubbing variables for example.

Reply:
List of variables updated